build/build.py

```diff
@@ -433,19 +436,28 @@ def main():
   )
   add_boolean_argument(
       parser,
-      "build_cuda_kernel_plugin",
+      "build_gpu_kernel_plugin",
```

Comment:
Wdyt about making this a str argument instead with `"cuda"`/`"rocm"` as the possible values?